apps/fz_common/test/fz_net_test.exs

```diff
@@ -148,4 +148,16 @@ defmodule FzCommon.FzNetTest do
       assert "2600:1:1:1:1:1:1:1" == FzNet.endpoint_to_ip("[2600:1:1:1:1:1:1:1]:4562")
     end
   end
+
+  describe "inet_to_ip_with_mask/2" do
+    test "with netmask" do
+      assert "10.3.2.1/24" ==
+               "#{FzNet.inet_to_ip_with_mask(%Postgrex.INET{address: {10, 3, 2, 1}, netmask: 24}, "10.3.2.0/24")}"
```

Comment:
A couple of small things here:
1. When we assert on static return it's better to keep the expected value on the right, it makes code less prone to typos when you use `=` instead of `==` and accidentally match on things instead of comparing them.
2. If we expect the function to return a string we should not cast it in the test, otherwise, we don't test for the return type.
```suggestion
      assert FzNet.inet_to_ip_with_mask(%Postgrex.INET{address: {10, 3, 2, 1}, netmask: 24}, "10.3.2.0/24") == "10.3.2.1/24"
```